### ISSUE TITLE
Add Playwright as an option?

### ISSUE THREAD

**Issue by yuqianma** [nolanlawson/fuite#44]
Our team use https://playwright.dev/ to run tests.

It saves some space and make things more seamless if fuite can reuse Playwright in the project.

Any plan?

**Comment by nolanlawson** [comment]
I haven't looked into it, but I can say that using Playwright to run non-Chromium projects would have low value, since fuite relies on Chromium-specific APIs (such as heap snapshots).

Could you provide more details on why it would be useful to use Playwright under the hood? Is it just that you want to run this on the same system that runs your existing tests, and your existing tests use Playwright, so it would use up less disk space?

**Comment by p01** [comment]
Exactly. Reducing the number of deps and disk usage for projects already using Playwright would be nice.

**Comment by yuqianma** [comment]
Almost forget this issue. Sorry for the delayed response.

I guess most of the Playwright users use it on top of Chromium-based browsers. Since they are all Chromium, fuite should be able to run in Playwright environment smoothly.

Besides disk usage, it also reduces the fragmentation of dependencies. If one integrates the fuite to CI, the same test env makes the workflow seamless.

**Comment by fcollonval** [comment]
I was looking at adding support for playwright. But there is a blocker for now: `page.queryObjects` is not available in Playwright.

Ref: https://github.com/microsoft/playwright/issues/14134
